pilot/pkg/proxy/envoy/v2/ads.go

```diff
@@ -148,7 +148,7 @@ var (
 	proxiesConvergeDelay = prometheus.NewHistogram(prometheus.HistogramOpts{
 		Name:    "pilot_proxy_convergence_time",
 		Help:    "Delay between config change and all proxies converging.",
-		Buckets: []float64{.01, .1, 1, 3, 5, 10, 30},
+		Buckets: []float64{1, 3, 5, 10, 20, 30, 50, 100},
```

Comment:
Our data so far shows that our bucketization was too fine grained below 30 and had no coverage above 30, which was a problem. @douglas-reid